doc/source/programs/gdal_raster_calc.rst

```diff
@@ -84,6 +84,27 @@ Program-Specific Options
     Arguments to functions are passed within parentheses, like ``sum(k=1)``,
     ``min(propagateNoData=true)`` or ``interpolate_linear(t0=1,dt=1,t=10)``.
 
+    Built-in variables
+    ^^^^^^^^^^^^^^^^^^
+
+    The following built-in variables are available in expressions:
+
+    - ``_CENTER_X_``: X coordinate of the pixel center, expressed in the dataset CRS
+    - ``_CENTER_Y_``: Y coordinate of the pixel center, expressed in the dataset CRS
+
+    These variables are useful for calculations that depend on spatial position,
+    such as latitude-based corrections, solar angle approximations, or zonal masks.
+
+    Example:
```

Comment:
Can you add this to the "Examples" section at the bottom and then cross-reference it here (with `:example:`, see `gdal_raster_mosaic.rst` for an example)